02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+            },
+            "Who are mandatory HH members for food stamps?": {
+                "score_sum": 0.6699078276008735,
+                "derived_questions": [
+                    "Does the grandchild live with you?",
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "Who are mandatory HH members for food stamps?\nAnswer:\nThe following household members are ALWAYS included, even if they report that they do not share/prepare meals together. (see text only below, see flow chart further down)\nSpouses:\nIf living in the same residence\nChildren:\nIf age 17 or younger, the child is mandatory with legal guardian\nIf age 18-21, the child is mandatory if natural, adopted, or step child*\nA mandatory child's own child or spouse would need to be included on the application as well\n*Ex. If a 20 year old child resides with their parent, they would be mandatory.\n*Ex. If a 20 year old child resides with their grandparent, they would NOT be mandatory. They would likely be sharing/preparing meals together, but are not required to be listed on the same SNAP application.\n\ufeff\n\ufeff",
+                "summary": "The grandchild living with the client should be included as a mandatory household member for food stamps, regardless of whether they share food or not."
+            },
+            "In what cases can a parent who has a joint custody arrangement include a child on a food stamp application?": {
+                "score_sum": 0.5218728563128504,
+                "derived_questions": [
+                    "Does the grandchild live with you?",
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "In what cases can a parent who has a joint custody arrangement include a child on a food stamp application?\nAnswer:\n(a) When the custody arrangement between parents is uneven, the child would be included in the household that provides the child with the majority of his or her meals.\n(b) When there is an equal custody arrangement and the children eat 50 percent of their meals with each parent, the case manager must decide which household should include the child. If both parents apply and cannot agree on which household will include the child, the case manager will resolve the issue by including the child in the household that applied first.\nNote: The case manager must insure that the child is included in only one household.\nFood Stamps only",
+                "summary": "The grandchild should be included in the household that provides the majority of their meals, or if there is an equal custody arrangement, the household that applied first."
```

Comment:
**Relevance to the question**: 🔴 Irrelevant
- Summary describes rules based on joint custody arrangements, which is irrelevant to the client situation where a 19yo grandchild lives with their grandparent
**Correctness as a summary of the Guru card**: 🟢 100% correct
**Helpfulness**: 🟠 Frustrating
- The summary is obviously irrelevant to the question, which is frustrating and confusing but not likely to result in the specialist going down the wrong path